axolotl-web/src/pages/Contacts.vue

```diff
@@ -43,11 +43,11 @@
                   alt="Avatar image"
                   @error="onImageError($event)"
                 />
-                {{ c.Name[0] ? c.Name[0] + c.Name[1] : c.Name[0] }}
+                {{ c.name[0] ? c.name[0] + c.name[1] : c.name[0] }}
```

Comment:
This line still contains a bug. Correct would be:
`{{ c.name[1] ? c.name[0] + c.name[1] : c.name[0] }}`

Also: I would like to introduce a function for this logic. I'm not a fan of logic within the template.